### ISSUE TITLE
Compatibility with `pytest=8.0.0`?

### ISSUE THREAD

**Issue by benjaminrigaud** [dev-petrov/pytest-lazy-fixtures#3]
Hi,

We cannot install the library with `pytest=8.0.0`

``` bash
pytest-lazy-fixtures 1.0.1 requires pytest<8.0.0,>=7.2.1, but you have pytest 8.0.0 which is incompatible.
```

But when forcing the installation, my test-suite is working fine.
Do you know of any limitations that would prevent it from working with `pytest=8.0.0`?

FYI: `pytest-lazy-fixture` itself is not compatible with pytest=8.0.0 (See https://github.com/TvoroG/pytest-lazy-fixture/issues/65)

**Comment by kjaymiller** [comment]
+1 for the same question for the exact same reason

**Comment by dev-petrov** [comment]
Hi, I will update dependency

**Comment by dev-petrov** [comment]
Fixed in [1.0.2](https://github.com/dev-petrov/pytest-lazy-fixtures/releases/tag/1.0.2)
